### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by jlberbesi**
Hello fellow code reviewer, I did the following tasks:

Implement the design from the sneak peek wireframes.
Styling is nice to have but focus on making sure everything functions first. (Note: Ignore login logic, you would be using Devise in coming milestone)

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1,6 +1,17 @@
...
+<h1>Users</h1>
+<ul class="users-list">
   <% @users.each do |user| %>
-    <li><%= user.name %></li>
+  <div class="list">
+    <div class="user-image">
+        <img src="path_to_default_user_image.jpg" alt="User Image">
+      </div>
+    <li class="user-container">
+
```
Comment (Munsa1) [inline app/views/users/index.html.erb]: - [ ] Kindly ensure you fix the path to the user Image for your views. I was able to add the image links but they were unable to be displayed. Kindly fix this.

Video demo
[cinnamon-2023-12-02T180232+0200.webm](https://github.com/jlberbesi/blog-app/assets/81932478/051a98b1-b690-4bdc-b688-5549018950a7)

File: app/views/posts/show.html.erb [hunk 2]
```diff
@@ -1,2 +1,18 @@
...
+  <h2>Comments</h2>
+    <% @post.comments.each do |comment| %>
+      <li>
+        <strong><%= comment.user.name %>:</strong>
+        <p><%= comment.text %></p>
+      </li>
+    <% end %>
+  </ul>
+  <%= link_to 'Back', request.referer || root_path %>
+</div>
```
Comment (Munsa1) [inline app/views/posts/show.html.erb]: - [ ] Kindly ensure the top part of the Post show page should follow the given wireframe.

Given design
image

Your design
image

- Ensure the comments and likes are aligned to the right as is in the wireframe

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @jlberbesi, 
required changes

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Passing Linters :heavy_check_mark: 
Followed Wireframe Design :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._
- [ ] Kindly remove the following links as they point to parts of the readme file that have been removed.
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/index.html.erb (hunk 1 above)
Comment (jlberbesi) [reply app/views/users/index.html.erb]: imagen
Imagen de WhatsApp 2023-12-02 a las 13 42 57_d9902d66
Imagen de WhatsApp 2023-12-02 a las 13 43 10_12e03a6d

File: app/views/posts/show.html.erb (hunk 2 above)
Comment (jlberbesi) [reply app/views/posts/show.html.erb]: Imagen de WhatsApp 2023-12-02 a las 13 36 30_81445d16

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @jlberbesi, 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Requested change added 🔥 


## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**